content/en/docs/setup/platform-setup/prerequisites/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Platform Prerequisites
+description: Prerequisites for platform setup for istio.
+weight: 1
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/prerequisites/
+    - /docs/setup/kubernetes/platform-setup/prerequisites/
+keywords: [platform-setup,prerequisites]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+
+## Kernel Module Requirements on Cluster Nodes
+
+The following kernel modules must be loaded on the cluster node running application pods with a Service Mesh proxy sidecar container.
+The modules are needed specifically by the `istio-init` container which sets up iptables rules in the pod to redirect any incoming or
+outgoing traffic towards the sidecar proxy in the istio-proxy container. While in many platforms, these seem to be automatically
+loaded, it is always good to make sure the prerequisites are met, as there were incidents reported where some of the specific
+modules listed down below were not loaded.
+
+| Module | Remark |
```

Comment:
nit: there is probably some TPROXY ones required for TPROXY mode. Not sure what the are.

What did you use to find out the full set used?

Reply:
Finding the full set was a manual effort, used the golden files at tools/istio-iptables/pkg/capture/testdata/ for input and then with the descriptions about module loading from the iptables manpage gathered all those that would be loaded based on the reference files.

Eg:

[pkg/capture/testdata]$ grep -hoP -e '-t \S+' * | sort | uniq
-t mangle
-t nat
-t raw